andela/travel_tool_back
# #160096049 Fix incorrect open requests badge count

#### What does this PR do?
Fix incorrect open requests badge count

#### Description of Task to be completed?
- Include the userId in the query to count user's open requests
- Include the userId in the query to count user's past requests
- Fix the failing tests

#### How should this be manually tested?
- Clone the repo -` git clone https://github.com/andela/travel_tool_back.git`
- Enter into the project directory - `cd travel_tool_back`
- Start the app using `make start`
- Open `Postman` and make a `POST` request to the endpoint `http://local.andela.com:5000/api/v1/requests` with the following fields `name`, `origin`, `destination`, `manager`, `gender`, `department`, `role`, `arrivalDate`, `departureDate`.
 (**Please see background context section**)
- Next, make a `GET` request to the same endpoint above
- You should see your request with the correct count.

#### Any background context you want to provide?
This endpoint requires authentication. To include token in the `authorization` header:
- Login to the [travela app](https://travela-staging.andela.com)
- After you are logged in, visit the developer console in your browser and open `Application`, you should see your `jwt-token` there.
- Copy the token and add it to the `authorization` header when making a request.


#### What are the relevant pivotal tracker stories?
[160096049](https://www.pivotaltracker.com/story/show/160096049)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-08-29 at 1 31 42 pm" src="https://user-images.githubusercontent.com/25195857/44787786-ff7f3580-ab8f-11e8-875e-56c1977efa02.png">
<img width="1440" alt="screen shot 2018-08-29 at 1 31 47 pm" src="https://user-images.githubusercontent.com/25195857/44787788-027a2600-ab90-11e8-89ad-1b1cd1b35acd.png">

#### Questions:
None
